src/engine/SCons/Script/Main.py

```diff
@@ -1099,17 +1100,36 @@ def _main(parser):
                                           targets, target_top)
 
     else:
```

Comment:
In general, we'd like to minimize our changes to the official SCons sources. So most of this code that deals with `vulcan` interfacing should move into the `Spirent` module.